23 -    fix: add breadcrumb block

Toto-Tuo opened the pull request:
src: https://github.com/hlxsites/elixirsolutions/tree/19da1f3448b7933d85f6c198d2b0293d55b339ce/blocks/breadcrumb
	new file:   blocks/breadcrumb/breadcrumb.css
	new file:   blocks/breadcrumb/breadcrumb.js
	modified:   scripts/scripts.js

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #23 - add-breadcrumb-block

Test URLs:
- Before: https://main--fresenius--hlxsites.hlx.page/
- After: https://23-add-breadcrumbs-block--fresenius--hlxsites.hlx.page/

Toto-Tuo committed:
fix: add breadcrumb block
    src: https://github.com/hlxsites/elixirsolutions/tree/19da1f3448b7933d85f6c198d2b0293d55b339ce/blocks/breadcrumb
	new file:   blocks/breadcrumb/breadcrumb.css
	new file:   blocks/breadcrumb/breadcrumb.js
	modified:   scripts/scripts.js

helms-charity commented on scripts/scripts.js:
```diff
@@ -1,6 +1,5 @@
 import {
   sampleRUM,
-  /* buildBlock, */
```
why did you delete that? I commented out temporarily but we'll use it soon enough and I didn't want to forget to add it here :-)

Toto-Tuo replied:
@helms-charity 
I added it back, I uncommented it first,  Lint wasn't passed, I tried to comment it out but deleted it by mistake.
Sorry about it.

Toto-Tuo commented:
@helms-charity 
I added the block but did not reference it in main scripts.js, when I tried to add it [0] in main script.js. Lint complained it was defined but never used. 

Could we add the reference later?

Thanks,

[0]
You will also need to reference the breadcrumb in styles.js, see here for example:
https://github.com/hlxsites/elixirsolutions/blob/main/scripts/scripts.js#L204

helms-charity reviewed (commented):
Can you uncomment the section in line 89-98 in scripts.js, and replace "buildHeroBlock(main);"
with
buildBreadcrumbBlock(main);

Then after you replace line #3, leave that uncommented too... and then your breadcrumbs should be added to this new page I created. https://main--fresenius--hlxsites.hlx.page/testpage when viewed via your branch.

Toto-Tuo committed:
add:  buildBreadcrumbBlock(main) func
    now Lint is not passing buildAutoBlocks (Ln 104, Col 10)
	modified:   scripts/scripts.js

Toto-Tuo commented:
@helms-charity 
I am stuck on above build error. I am going to reach out to you for help tomorrow.
Thank you so much :)

helms-charity reviewed (commented):
Oops, also uncomment line 122 :-)

helms-charity committed:
fix: uncommented last function for autoblock build

helms-charity reviewed (approved):
I did the last uncomment, and I can see your breadcrumbs on https://23-add-breadcrumbs-block--fresenius--hlxsites.hlx.page/testpage :-)

Toto-Tuo commented:
That was it! Thank you @helms-charity :-) !!!
